src/eredis_cluster.erl

```diff
@@ -843,3 +846,35 @@ get_key_from_rest([_, KeyName|_]) when is_list(KeyName) ->
     KeyName;
 get_key_from_rest(_) ->
     undefined.
+
+-ifdef(TEST).
+-include_lib("eunit/include/eunit.hrl").
+
+parse_redirect_info_test() ->
+    %% Address and port can be parsed
+    {error, no_pool} = parse_redirect_info(<<"1 127.0.0.1:12345">>),
+    {error, no_pool} = parse_redirect_info(<<"1 ::1:12345">>),
```

Comment:
Good test case!

I guess it's safe to assume that if it returns no_pool, it has managed to split it at the right colon, convert the port to an integer and validate the address. Nice.